test/indexTest.js

```diff
@@ -0,0 +1,11 @@
+var assert = require("assert")
+var server = require("./testServer")
```

Comment:
I think the only conflict would be the port — I'll update mine so the test env uses a different one.